ddp-workspace/projects/ddp-atcp/src/theme.scss

```diff
@@ -0,0 +1,120 @@
+@import '../../../node_modules/@angular/material/theming';
+@include mat-core();
+
+$md-primary: (
+    50 : #eeeaff,
+    100 : #d4ccff,
+    200 : #b8aaff,
+    300 : #9c87ff,
+    400 : #866eff,
+    500 : #7154ff,
+    600 : #694dff,
+    700 : #5e43ff,
+    800 : #543aff,
+    900 : #4229ff,
+    A100 : #ffffff,
+    A200 : #ffffff,
+    A400 : #dad6ff,
+    A700 : #c3bdff,
```

Comment:
Looks like you need to update colors palette, because material components use Osteo color:
![image](https://user-images.githubusercontent.com/30126907/79974548-33644880-84a2-11ea-9a71-0a0f28bddb05.png)

Reply:
You're right. I've used Osteo study as a base for AT-Study and it looks like some Osteo artefacts left. Will fix it